src/app/statistics-page/statistics-table/statistics-table.component.spec.ts

```diff
@@ -97,4 +97,31 @@ describe('StatisticsTableComponent', () => {
         .toEqual('8');
     });
   });
+
+  describe('getObjectHeaderLabel', () => {
+
+    it('should return the correct label for TotalVisits', () => {
+      expect(component.getObjectHeaderLabel('TotalVisits')).toEqual('statistics.table.header.TotalVisits');
+    });
+
+    it('should return the correct label for TotalVisitsPerMonth', () => {
+      expect(component.getObjectHeaderLabel('TotalVisitsPerMonth')).toEqual('statistics.table.header.TotalVisitsPerMonth');
+    });
+
+    it('should return the correct label for TotalDownloads', () => {
+      expect(component.getObjectHeaderLabel('TotalDownloads')).toEqual('statistics.table.header.TotalDownloads');
+    });
+
+    it('should return the correct label for TopCities', () => {
+      expect(component.getObjectHeaderLabel('TopCities')).toEqual('statistics.table.header.TopCities');
+    });
+
+    it('should return the correct label for topCountries', () => {
+      expect(component.getObjectHeaderLabel('topCountries')).toEqual('statistics.table.header.topCountries');
+    });
+
+    it('should return an empty string for unknown report types', () => {
+      expect(component.getObjectHeaderLabel('UnknownType')).toEqual('');
```

Comment:
This is where your tests are failing.  It looks like you previously were expecting `getObjectHeaderLabel()` to return an empty string if you passed it an unrecognized type.  

This isn't going to work the same anymore, because you are always just returning `statistics.table.header.[type]` in the code in `getObjectHeaderLabel()`.  So, I think you can just remove this test... all types will now be looked up in the i18n files.  This seems appropriate though because it allows for adding new report types without having to modify your code.